langchain/tools/python/tool.py

```diff
@@ -28,13 +28,13 @@ class PythonREPLTool(BaseTool):
     python_repl: PythonREPL = Field(default_factory=_get_default_python_repl)
     sanitize_input: bool = True
 
-    def _run(self, query: str) -> str:
+    def _run(self, tool_input: str) -> str:
```

Comment:
Might this be a breaking change for people who have written custom tools

Reply:
good pt, i can look into making it less breaking